### PR TITLE
BNGP-4472: Update Dashboard "Manage Biodiversity Gains" page to include the credits service option

### PR DESCRIPTION
**Pull request by usman-j-shaikh**

### CODE REVIEW

File: packages/webapp/src/utils/constants.js [hunk 1]
```diff
@@ -50,10 +51,12 @@ const ORGANISATION = 'organisation'
 const MULTIPLE_PROOFS_OF_PERMISSION_REQUIRED = 'multipleProofsOfPermissionRequired'
 const ACCESSIBILITY_STATEMENT = 'accessibility-statement'
 const COOKIES = 'cookies'
+const CREDITS_PURCHASE = 'credits-purchase'
```
Comment (paulsimonandrews) [inline packages/webapp/src/utils/constants.js:54]: Change to `CreditsPurchase` as that is what the type is called in the database.

File: packages/webapp/src/routes/manage-biodiversity-gains.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import { getCreditsRedirectURL } from '../utils/helpers.js'
 
 const handlers = {
   get: async (request, h) => {
     const { representing } = getApplicantContext(request.auth.credentials.account, request.yar)
     return h.view(constants.views.MANAGE_BIODIVERSITY_GAINS, {
       representing,
-      enableDev: process.env.ENABLE_ROUTE_SUPPORT_FOR_DEV_JOURNEY === 'Y'
+      enableDev: process.env.ENABLE_ROUTE_SUPPORT_FOR_DEV_JOURNEY === 'Y',
+      enableCredits: process.env.ENABLE_ROUTE_SUPPORT_FOR_CREDIT_ESTIMATION_JOURNEY === 'Y',
```
Comment (paulsimonandrews) [inline packages/webapp/src/routes/manage-biodiversity-gains.js:11]: Should be `ENABLE_ROUTE_SUPPORT_FOR_CREDIT_PURCHASE_JOURNEY`

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Couple of minor changes to get it working correctly

**Review by paulsimonandrews — APPROVED** [review]
LGTM

**Comment by shaundefra** [comment]
tested locally

**Pull request merged**